boards/xtensa/intel_s1000_crb/doc/index.rst

```diff
@@ -125,6 +125,12 @@ software keys you have downloaded from `Tensilica Tools for Sue Creek`_
    follow what the license manager expects or create a dedicated ethernet device
    for this to work.
 
+   On the latest LTS ubuntu(18.04), you can install the license key manually with
+
+.. code-block:: console
```

Comment:
indentation is off, this needs to be part of the node.

Reply:
Hi Anas, it does not seems to me that we can turn on indentation on this line. I tried indenting the line. Then the line of "cp...." did not show up as a code-block.

Reply:
Hi Anas, please ignore my comments above. You are right. David told me the detail of how to fix it. Please take look at commit 28a98da.